index.js

```diff
@@ -12,8 +12,46 @@ var unlink = RSVP.denodeify(fs.unlink);
 var tmpDir = require('os').tmpdir();
 var debug = require('debug')('async-disk-cache');
 var zlib = require('zlib');
+var heimdall = require('heimdalljs');
 
 var CacheEntry = require('./lib/cache-entry');
+var Metric = require('./lib/metric');
+
+if (!heimdall.hasMonitor('async-disk-cache')) {
```

Comment:
Using this guard at module level indicates you expect different versions of this module to be required. If that is the case, this is alright, but if not this is an unnecessary and potentially even problematic check.